src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope_PAL.c

```diff
@@ -490,26 +529,29 @@ void KaleidoScope_SetDefaultCursor(PlayState* play) {
     }
 }
 
+#define SWITCH_PAGE_LEFT_PT 0
+#define SWITCH_PAGE_RIGHT_PT 2
```

Comment:
What's the `PT` here?

Reply:
It comes from this debug string

https://github.com/zeldaret/oot/blob/b3486b57ef41a284a366eb40ca07bf82bf4750d6/src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope_PAL.c#L514

PT values are for the pt variable

I guess I could rename it to direction or something maybe?

-----

Pure guess with no proof: maybe "pt" stands for "page turn" 8)